sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -148,11 +148,16 @@ static void send_wp_moved(struct transport_tx *trans, struct link_device *dev)
   static uint8_t i;
   i++;
   if (i >= nb_waypoint) { i = 0; }
+  struct EnuCoor_i wp_i;
```

Comment:
That's an artefact of the project I am actually working on, wherefore I needed to increase INT32_POS_FRAC. Then, this was to ensure the telemetry precision was still 8 bits as defined in messages.xml.